flycheck.el

```diff
@@ -7774,24 +7762,24 @@ enabled. "
   :safe #'flycheck-string-list-p)
 
 (flycheck-define-checker go-vet
-  "A Go syntax checker using the `go tool vet' command.
+  "A Go syntax checker using the `go vet' command.
 
 See URL `https://golang.org/cmd/go/' and URL
 `https://golang.org/cmd/vet/'."
-  :command ("go" "tool" "vet" "-all"
-            (option "-printfuncs=" flycheck-go-vet-print-functions concat
+  :command ("go" "vet"
+            (option "-printf.funcs=" flycheck-go-vet-print-functions concat
                     flycheck-option-comma-separated-list)
-            (option-flag "-shadow" flycheck-go-vet-shadow)
-            (option-list "-tags=" flycheck-go-build-tags concat)
-            (eval (when (eq flycheck-go-vet-shadow 'strict) "-shadowstrict"))
+            ;; (option-flag "-shadow" flycheck-go-vet-shadow) ;; DEPRECATED
+            ;; (option-list "-tags=" flycheck-go-build-tags concat) ;; DEPRECATED
+            ;; (eval (when (eq flycheck-go-vet-shadow 'strict) "-shadowstrict")) ;; DEPRECATED
             source)
   :error-patterns
   ((warning line-start (file-name) ":" line ": " (message) line-end))
   :modes go-mode
-  ;; We must explicitly check whether the "vet" tool is available
+  ;; We must explicitly check whether the "vet" command is available
```

Comment:
I thought `go vet` was always present as long as `go` was installed?

Reply:
Unless your go package is broken, I guess. 

Should I remove it or just leave it in as a check that go is install properly?

Reply:
We already checks the `:command` part exists, you can remove the `:predicate` and `:verify` which are redundant then.